subprojects/docs/src/docs/release/notes.md

```diff
@@ -104,6 +104,28 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
+<a name="dependency-verification"></a>
+
+### Dependency verification improvements
+
+When Gradle is asked to export all trusted keys via `./gradlew --export-keys`, it generates both binary and ASCII armored versions of the keys.
+With this release you can change this behavior and choose only one format.
+In order to do so, you can edit `verification-metadata.xml` by adding the `keyring-format` setting.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<verification-metadata xmlns="https://schema.gradle.org/dependency-verification"
+                       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                       xsi:schemaLocation="https://schema.gradle.org/dependency-verification https://schema.gradle.org/dependency-verification/dependency-verification-1.3.xsd">
+    <configuration>
+        <verify-metadata>true</verify-metadata>
+        <verify-signatures>true</verify-signatures>
+        <keyring-format>text</keyring-format> <!-- or gpg -->
+    </configuration>
+</verification-metadata>
+```
+
+We've also updated the XML validation schema for `verification-metadata.xml` to be more concise.
```

Comment:
Could be nice to get a little more information or an example of this. How is it more concise?